install/kubernetes/helm/istio-remote/templates/service.yaml

```diff
@@ -60,3 +60,44 @@ spec:
     protocol: UDP
   clusterIP: None
 ---
+apiVersion: v1
+kind: Service
+metadata:
+  name: istio-telemetry
+  namespace: istio-system
+spec:
+  clusterIP: 10.100.243.220
+  ports:
+  - name: http2-mixer
+    port: 9091
+    protocol: TCP
+    targetPort: 9091
+  - name: tcp-mtls
+    port: 15004
+    protocol: TCP
+    targetPort: 15004
+  - name: http-monitoring
+    port: 9093
+    protocol: TCP
+    targetPort: 9093
+  - name: prometheus
+    port: 42422
+    protocol: TCP
+    targetPort: 42422
+  clusterIP: None
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: istio-ingress
+  namespace: istio-system
+spec:
+  ports:
+  - name: http
+    port: 80
+    protocol: TCP
+  - name: https
+    port: 443
+    protocol: TCP
+  clusterIP: None
```

Comment:
conditionalize these please such as:
https://github.com/istio/istio/blob/master/install/kubernetes/helm/istio-remote/values.yaml#L5-L13

and:

https://github.com/istio/istio/blob/master/install/kubernetes/helm/istio-remote/templates/endpoints.yaml#L8

Reply:
Not sure I understand your comments. it's intentional to make the clusterIP as None.

Reply:
Ahh - ok makes sense.  Although further up, there is an IP endpoint specified which is not conditionalized.